src/screens/NetworkSettings/NetworkSettingsDisclaimerScreen.js

```diff
@@ -30,6 +31,10 @@ const style = StyleSheet.create({
 export const NetworkSettingsDislaimerNav = Symbol('NetworkSettingsDislaimer').toString();
 
 export function NetworkSettingsDislaimerScreen({ navigation }) {
+  const handleNavigation = () => {
+    navigation.push(NetworkPreSettingsNav);
+  };
```

Comment:
Can we do this?
```suggestion
  const handleNavigation = () => navigation.push(NetworkPreSettingsNav);
```

Reply:
- Change lambda to inline expression: c9bee3624619955be9868f29a21e98312a37cd7a